docs/releases/2.1.1.txt

```diff
@@ -18,3 +18,6 @@ Bugfixes
 * Fixed a regression in Django 2.0 where using ``manage.py test --keepdb``
   fails on PostgreSQL if the database exists and the user doesn't have
   permission to create databases (:ticket:`29613`).
+
+* Fixed a SQL compiler bug when using subqueries within methods like
+  ``TruncMonth()`` (:ticket:`29648`).
```

Comment:
From what I can see this isn't a regression in 2.1 and thus doesn't qualify for a backport.